src/dialects/postgres/query-generator-typescript.ts

```diff
@@ -75,4 +75,27 @@ export class PostgresQueryGeneratorTypeScript extends AbstractQueryGenerator {
       options?.cascade ? 'CASCADE' : '',
     ]);
   }
+
+  /**
+   * Generates an SQL query that returns all foreign keys of a table.
+   *
+   * @param   tableName The table or associated model.
+   * @returns           The generated SQL query.
+   */
+  // TODO: allow schema to be specified here as well
```

Comment:
Isn't this done? The schema can be specified as part of the `tableName` option

Reply:
Yes, but it's not used in the resulting query. Using `mySchema.myTable` for example does not work so we need to update the query to provide the schema in the correct spot

Reply:
here you go

```sql
AND r.connamespace = (SELECT oid FROM pg_namespace WHERE nspname = 'public' LIMIT 1)
```

Reply:
Thanks!

Reply:
Let's leave this test for another PR.
`getForeignKeyQuery` is only used in `getForeignKeysForTables` for all dialects. That in return is only used by `dropAllTables` (for all dialects expect for sqlite). That uses `showAllTables` which uses the 'database' from the config. `getForeignKeysForTables` only accepts a string array of table names from that and discards the schema. So basically we need to improve `getForeignKeysForTables` and `showAllTables` to deal with schemas as well and then we can return to this test.

Alternative is that we just run the query from `getForeignKeyQuery` raw and check it that way

Which one do you prefer?

Reply:
Let's wait until we've improved the query interface method then